components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/template/APITemplateBuilderImpl.java

```diff
@@ -41,65 +49,100 @@
 
 	private static final Log log = LogFactory.getLog(APITemplateBuilderImpl.class);
 
-	public static final String TEMPLATE_TYPE_VELOCITY = "velocity_template";
+	private static final String VELOCITY_TEMPLATE_SYNAPSE_CONFIG_NON_VERSIONED_WEBAPP =
+			"velocity-template_synapse-config_non-versioned-webapp.xml";
+	private static final String VELOCITY_TEMPLATE_SYNAPSE_CONFIG_VERSIONED_WEBAPP =
+			"velocity-template_synapse-config_versioned-webapp.xml";
 
-    private WebApp api;
-    private List<HandlerConfig> handlers = new ArrayList<HandlerConfig>();
+	private WebApp api;
```

Comment:
fixed in commit 533c80fbc60e2610e57e9087ae720bd682fa6fc9 of PR https://github.com/wso2/carbon-appmgt/pull/153